app/stac_api/models.py

```diff
@@ -111,6 +113,21 @@ def validate_item_properties_datetimes_dependencies(
 	Raises:
 		django.core.exceptions.ValidationError
     '''
+    try:
+        if not isinstance(properties_datetime, datetime) and properties_datetime is not None:
+            properties_datetime = fromisoformat(properties_datetime)
+        if not isinstance(
+            properties_start_datetime, datetime
+        ) and properties_start_datetime is not None:
+            properties_start_datetime = fromisoformat(properties_start_datetime)
+        if not isinstance(
+            properties_end_datetime, datetime
+        ) and properties_end_datetime is not None:
```

Comment:
Just minor code style comment, having `if .. and ..:` enclosed in brackets make the code easier to read; 
```
if (not isinstance(properties_start_datetime, datetime) and
    properties_start_datetime is not None
):
```
Yapf does a better wrapping with brackets as without.